spec/builtin-completion.test.sh

```diff
@@ -162,6 +162,37 @@ PATH
 PWD
 ## END
 
+#### compgen -e with global/local exported vars
+export v1_global=0
+f() {
+  export local v2_local=0   
```

Comment:
nitpick : should be

```
local v2_local=0
export local
```

export local actually exports a variable named "local"!

Reply:
Sorry about that. I'm not much of a shell scripter. 

I'm more the sort who liked your blog posts, thought I'd give osh a try, and have just been trying to get this one thing working so I can test it as a daily driver :P 

Wait, should that not be `export v2_local` instead of `export local`?

Reply:
Oh yes, typo!   the var name comes after export

No problem, it is easy to make these mistakes, hence Oils :)